examples/headsail-bsp/src/sysctrl/mmap.rs

```diff
@@ -3,6 +3,7 @@
 /// SysCtrl-specific, SysCtrl internal memory maps
 
 pub(crate) const SYSCTRL_ADDR: usize = 0x1a10_0000;
+pub(crate) const SDRAM_BASE_ADDR: usize = 0xFFD00000;
```

Comment:
This doesn't look like a SysCtrl specific memory map. Is it global? If so, please move it to src/mmap.rs.

Reply:
Oh, this is also not the SDRAM base but rather the SDRAM **config** base in the global map (ref. https://soc-hub.gitlab-pages.tuni.fi/headsail/hw/headsail/address_map.html). Please rename to `SDRAM_CONFIG_ADDR`  for consistency.

Reply:
Name changed to `SDRAM_CONFIG_ADDR` and moved out of SysCtrl internal memory map to the global memory map.